examples/sandbox/index.ts

```diff
@@ -1819,6 +1822,36 @@ $ethEIP1559.on("click", async () => {
   ethEIP1559Selected = !ethEIP1559Selected;
 });
 
+$ethSignTypedData.on("click", async (e) => {
+  e.preventDefault();
+  if (!wallet) {
+    $ethResults.val("No wallet?");
+    return;
+  }
+  if (core.supportsETH(wallet)) {
+    const result = await wallet.ethSignTypedData({
+      ...eip712["calculateHashesOnDevice"],
```

Comment:
will have to update the sandbox payloads for `hashableData` as well so we don't forget